### PR TITLE
Add GraalVM for JDK 17 / JDK 20 casks.

### PR DESCRIPTION
**Pull request by fniephaus**

### CODE REVIEW

File: Casks/graalvm-jdk17.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+cask "graalvm-jdk17" do
+  arch arm: "aarch64", intel: "x64"
+
+  version "17.0.7"
+  sha256 arm:   "cb45f6585ef02134a6a6ffb6de20db96197486ffef8821ad97b11fe2fc0c23b8",
+         intel: "905255762546c69e3bb8d815a5d20e2e3cfa5332b868ab90af7aa0afe21e74ea"
+
+  installation_dir = "graalvm-jdk-#{version}+8.1".freeze
```
Comment (dougxc) [inline Casks/graalvm-jdk17.rb:8]: Where does `8.1` come from?

Comment (fniephaus) [reply Casks/graalvm-jdk17.rb:8]: This is the build number (JDK dot GraalVM).

Comment (dougxc) [reply Casks/graalvm-jdk17.rb:8]: JDK == 8?

Comment (fniephaus) [reply Casks/graalvm-jdk17.rb:8]: 8 is the JDK build number (17.0.7+**8**), 1 is the GraalVM build number.

**Comment by Techcable** [comment]
Why are you changing the cask names from `graalvm-ce` to `graalvm-community`?

If you're concerned about keeping both versions, wouldn't it be better to use a versioned cask?

**Pull request merged**